Sources/SwiftSyntaxBuilderGeneration/main.swift

```diff
@@ -0,0 +1,199 @@
+                       modifiersBuilder: {
+            if let text = token.text {
+              TokenSyntax.static.withLeadingTrivia(.newlines(1) + .docLineComment("/// The `\(text)` token"))
+            } else {
+              TokenSyntax.static
+            }
+          },
+                       bodyBuilder: {
+            FunctionCallExpr(calledExpression: MemberAccessExpr(base: "SyntaxFactory", name: "make\(token.name)"), leftParen: .leftParen, rightParen: .rightParen, trailingClosure: nil, argumentListBuilder: {
+              TupleExprElement(expression: IdentifierExpr("text"))
+            })
+
+            if token.requiresLeadingSpace {
+              FunctionCallExpr.leadingTriviaFunction
+            }
+
+            if token.requiresTrailingSpace {
+              FunctionCallExpr.trailingTriviaFunction
+            }
+          })
+        }
+      }
+      VariableDecl(letOrVarKeyword: .var,
+                   modifiersBuilder: { TokenSyntax.static.withLeadingTrivia(.newlines(1) + .docLineComment("/// The `eof` token") + .newlines(1)) },
+                   bindingsBuilder: {
+        // We need to use `CodeBlock` here to ensure there is braces around.
+        let body = CodeBlock {
+          FunctionCallExpr(calledExpression: MemberAccessExpr(base: "SyntaxFactory", name: "makeToken"),
+                           leftParen: .leftParen,
+                           rightParen: .rightParen,
+                           argumentListBuilder: {
+            TupleExprElement(expression: FunctionCallExpr(calledExpression: MemberAccessExpr(name: "eof")), trailingComma: .comma)
+            TupleExprElement(label: TokenSyntax.identifier("presence"), colon: .colon, expression: FunctionCallExpr(calledExpression: MemberAccessExpr(name: "present")))
+          })
+        }
+
+        PatternBinding.createTokenSyntaxPatternBinding("eof", accessor: body)
+      })
+      VariableDecl(letOrVarKeyword: .var,
+                   modifiersBuilder: { TokenSyntax.static.withLeadingTrivia(.newlines(1) + .docLineComment("/// The `open` contextual token") + .newlines(1)) },
+                   bindingsBuilder: {
+        // We need to use `CodeBlock` here to ensure there is braces around.
+        let body = CodeBlock {
+          FunctionCallExpr(calledExpression: MemberAccessExpr(base: "SyntaxFactory", name: "makeContextualKeyword"),
+                           leftParen: .leftParen,
+                           rightParen: .rightParen,
+                           argumentListBuilder: {
+            TupleExprElement(expression: StringLiteralExpr("open"))
+          })
+
+          FunctionCallExpr.trailingTriviaFunction
+        }
+
+        PatternBinding.createTokenSyntaxPatternBinding("open", accessor: body)
+      })
+    })
+}
+
+let tokenSyntax = tokensFile.buildSyntax(format: format)
+
+let arguments = CommandLine.arguments
+  .filter { $0.starts(with: "--") }
+  .map { $0.split(separator: "=").map { String($0) } }
```

Comment:
What do you think about just using the first argument as the destination? That would remove any need for command line argument parsing here. I.e. `let destination = CommandLine.arguments[1]` (plus some checking that we actually have a first argument).

Also, in the case that `destination` is not passed, we should output a little help message with the arguments SwiftSyntaxBuilderGenerat/main.swift expects.